collectors/charts.d.plugin/Makefile.am

```diff
@@ -31,7 +31,6 @@ dist_charts_DATA = \
 
 userchartsconfigdir=$(configdir)/charts.d
 dist_userchartsconfig_DATA = \
-    .keep \
     $(NULL)
```

Comment:
I'm not really an expert on automake, but do we need this variable at all anymore if it ends up being empty? Same for all the other changes like this in other files.

Reply:
You're probably right; but as neither of us are experts in `autotools` or C for that matter I'll just leave this alone for now :)